mappings/none/BlockInWorld.mapping

```diff
@@ -0,0 +1,19 @@
+CLASS none/asp none/BlockInWorld
```

Comment:
I don't think I explained this properly.

Depending on liveState it can either be set to pass-through to the world showing the current state, or show the cached state.

If it was just lazily setting it to a cached state I could be happy with calling it a blockSnapshot, but it acts differently.